### ISSUE TITLE
Testing: Backport changes in testsuite from rel-10_1 to rel-10_0

### ISSUE THREAD

**Issue by bschmalhofer** [RotherOSS/otobo#447]
For the 10.0.4 release it would be nice to have a test suite that works with prove and that provides a more meanigful test summary. 
TODO:

- [x] Migrate the adaptions to the test suite from 10_1 to 10_0, after the release of 10.0.3
- [x] Verify that the test suite runs 
- [x] Fix or skip tests that are specific to rel-10_1, or create in issue

**Comment by bschmalhofer** [comment]
Backporting should only be done after #192 is working in rel-10_1.

**Comment by bschmalhofer** [comment]
Backporting is done. issues for remaining failures are created. Closing this issue.
